Swift/BRCrypto/BRCryptoWalletManager.swift

```diff
@@ -152,20 +165,25 @@ public final class WalletManager: Equatable {
         self.query   = system.query
 
         self.defaultNetworkFee = network.minimumFee
+
+        self.addressSchemes = WalletManager.defaultAddressSchemes (currency: network.currency)
```

Comment:
Should "supported address schemes" be at the C level? Probably should for modes too, it occurs to me now.